### DISCUSSION TITLE
Introducing self-hosting AppFlowy with AppFlowy Cloud ☁️

### DISCUSSION THREAD

**Discussion by annieappflowy** [AppFlowy-IO/AppFlowy#4079] [General]
Hello @everyone, we're thrilled to introduce self-hosting AppFlowy with AppFlowy Cloud, designed for easy deployment and self-managed cloud storage. It can be hosted with cloud services such as Amazon EC2 and Azure Virtual Machines. For more information and setup instructions, please check out our guide: https://docs.appflowy.io/docs/guides/appflowy/self-hosting-appflowy. We welcome your feedback in the ⁠self-host-af-cloud channel!

🎁  Also, we're eager to gather valuable feedback from our users to better plan for 2024. Participants who complete this survey will gain priority access to our new launches. We appreciate your insights: https://tally.so/r/nW8qAQ.

**Comment by Andrew-Chen-Wang** [comment]
Congrats. Amazing work!

**Comment by arsen-z** [comment]
Great work! Is it possible to use other means of authentication than Google/GitHub/Discord? E.g., is it possible to set up a self-hosted Keycloak server and use it to authenticate? Thanks for the information!

**Reply by argseby** [reply]
@annieappflowy any updates on this? It would be nice to set up a user/password on creation and then authenticate the cloud with that. For a single user the use of other sign in options feels unnecessary and this leaves the application open to people applying to get an account.

**Reply by annieappflowy** [reply]
We now support magic link authentication. The next one will be with a password.

**Reply by lokesh-krishna** [reply]
Where can one track progress on password authentication or be notified when it's available?
